initialize.sh

```diff
@@ -92,7 +91,7 @@ if ! [ -x "$(command -v docker-compose)" ]; then
     exit 1
   fi
 else
-  IFS=',' read -ra temp <<< "$(docker-compose --version)"
+  IFS=',' read -ra temp <<< "$(docker compose version | cut -d 'v' -f3)"
```

Comment:
if you can launch `docker compose` it means that the version of docker compose plugin that you have installed is more recent than the minimum [required](https://intelowl.readthedocs.io/en/latest/Installation.html#requirements) (only from docker 2 there is this syntax). So it does not make sense to change this in my opinion.

This script should support both the existence of an old docker-compose and a new one, not just one of them.

The flow I think should be:
* verify that `docker compose` command run
* if yes, ok. If not, try `docker-compose` command run
* if yes, check version. If not, install `docker-compose` (this is already implemented)

Reply:
Yes it should support both  I'll change it